src/AccessibilityInsights.SharedUx/Controls/CustomControls/ScannerResultCustomListContext.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
```

Comment:
This file needs the license header 
```suggestion
// Copyright (c) Microsoft. All rights reserved.
// Licensed under the MIT license. See LICENSE file in the project root for full license information.
﻿
﻿﻿﻿using System;
```